src/main/java/org/jabref/gui/maintable/columns/FileColumn.java

```diff
@@ -139,7 +139,13 @@ private void setCommonSettings() {
 
     private String createFileTooltip(List<LinkedFile> linkedFiles) {
         if (!linkedFiles.isEmpty()) {
-            return Localization.lang("Open file %0", linkedFiles.getFirst().getLink());
+            StringBuilder tooltipText = new StringBuilder();
+
+            for (LinkedFile linkedFile : linkedFiles) {
+                tooltipText.append(linkedFile.getLink()).append("\n");
+            }
+
+            return Localization.lang("Open files: \n%0", tooltipText.toString());
```

Comment:
No need to include `\n` in the translation here.

```suggestion
            return Localization.lang("Open file(s): %0", tooltipText.toString());
```

In JabRef, we do pluralization with `(s)`. Reasoning (more or less) at https://phrase.com/blog/posts/pluralization/.